packages/ui-keyring/src/account/save.ts

```diff
@@ -13,4 +13,5 @@ export default function saveAccount (state: State, pair: KeyringPair, password?:
   }
 
   state.accounts.add(json.address, json);
+  state.keyring.addFromJson(json);
```

Comment:
Reasoning to this change?

Reply:
Without that change the user will encounter an error `ExtError: No encrypted data available to decode` when they try to change their password immediately after creating an account (unless of course they refresh the page after creating the account).

* Remove all accounts
* Create a new account with password "me"
* Click "Change Password"
* Enter the password "me"
* Enter the new password "me2"
* Click "Submit"
* View the error shown in the modal, which was encountered in the catch block of changePassword function in ui-keyring. It should show in "red" between the input fields and the buttons `ExtError: No encrypted data available to decode`

On a side note, would you like me to show such error notification in the UI as I've done. I thought this would be easier for users to report bugs and less load on tech support in production, as they could just submit a screenshot or copy/paste the error, and we wouldn't have to explain to them how to open Chrome Inspector to access the actual error if I did a console.error or were to throw the error instead.